docs/language/ql-training/_static-training/slides-semmle-2/static/theme/css/default.css

```diff
@@ -1667,6 +1667,15 @@ li > ul > li {
   margin-bottom: 0;
 }
 
+.admonition.note ol {
```

Comment:
This CSS tweak is required to make numbered lists look sensible on the notes pages.